public/src/pages/StudentPage.tsx

```diff
@@ -293,7 +305,7 @@ export class StudentPage extends ViewPage {
     private selectAssignment(labId: number) {
```

Comment:
Maybe `selectAssignment` and `selectGroupAssignment` can be combined into one function.

Reply:
Easy.